app/routes/telemetry/index.js

```diff
@@ -47,9 +49,19 @@ const CORE_METRICS = {
     "hydrocatFluorescence",
     "hydrocatTurbidity",
     "hydrocatPH",
-  ],
-  Hydrocycle: ["CAPO4"],
-  SUNA: ["sunaNitrateMicroMol"],
+    "hydrocatStartTime"],
```

Comment:
why are some `Start` and some `StartTime`? It would be easier to work with the fields if they were consistently `<table name in lowercase>StartTime` (or Start, but consistent across all tables)